pettingzoo/sisl/multiwalker/multiwalker_base.py

```diff
@@ -692,6 +688,7 @@ def render(self, close=False):
         self.surf = pygame.transform.flip(self.surf, False, True)
         self.screen.blit(self.surf, (-self.scroll * render_scale - offset, 0))
         if self.render_mode == "human":
+            pygame.event.pump()  # macos pygame window popup fix
```

Comment:
Let’s try to be consistent and always do pump before flip and on the same lines ( with or without comments, ppl can see the git blame if they really want to know why it’s there, but you can leave the comments if you want)